config/locales/generated/pt-BR.yml

```diff
@@ -1,14 +1,14 @@
-# This file has been generated by script/i18n/generate_languages_translations.
+# This file has been generated by ./script/i18n/generate_languages_translations.
 # Please do not edit directly.
 #
-# To update this file, run script/i18n/generate_languages_translations.
+# To update this file, run ./script/i18n/generate_languages_translations.
 #
 # The translations come from version 42 of the Unicode CLDR project .
 #
 # The Unicode Common Locale Data Repository (CLDR) provides key building
 # blocks for software to support the world's languages, with the largest
 # and most extensive standard repository of locale data available.
 ---
-pt:
+pt-BR:
   cldr:
     language_name: Português
```

Comment:
It feels wrong. It should mention Brazil somewhere.

Actually, in the CLDR, there is no language information for pt-BR from file `common/main/pt_BR.xml`, and the way the script works fall back to `pt` locale from `common/main/pt.xml` file instead. That's why it's "Português".

The translations for variations "pt-BR" and "pt-PT" do not exist in `common/main/pt.xml`, but do exist in `common/main/pt_PT.xml`:

```
  <language type="pt_BR">português do Brasil</language>
  <language type="pt_PT">português europeu</language>
```

The correct language name should be "Português do Brasil".

We could also use the "territory" information from `common/main/pt.xml` to generate the names:
```
  <territory type="BR">Brasil</territory>
  ...
  <territory type="PT">Portugal</territory>
```

Names could then be "Português (Brasil)" and "Português (Portugal)".

Which one would be better?

The script should be modified to be better at automatically resolving those edge cases.

Reply:
I modified the script to use "Português do Brasil" instead.